### PR TITLE
Updated files documentation

### PR DESCRIPTION
**Pull request by umbreak**
files-final

### CODE REVIEW

File: docs/src/main/paradox/docs/delta/api/current/assets/files/create-post.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+curl -X POST \
+   -F "file=@/path/to/myfile.jpg;type=image/jpeg" \
```
Comment (imsdu) [inline docs/src/main/paradox/docs/delta/api/current/assets/files/create-post.sh:2]: Align with the first line

File: docs/src/main/paradox/docs/delta/api/current/assets/files/create-put.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+curl -X PUT \
+   -F "file=@/path/to/myfile.pdf;type=application/pdf" \
```
Comment (imsdu) [inline docs/src/main/paradox/docs/delta/api/current/assets/files/create-put.sh:2]: Same

File: docs/src/main/paradox/docs/delta/api/current/kg-files-api.md [hunk 3]
```diff
@@ -23,9 +17,12 @@ Please visit @ref:[Authentication & authorization](authentication.md) section to
 ## Create a file using POST
```
Comment (imsdu) [inline docs/src/main/paradox/docs/delta/api/current/kg-files-api.md:17]: Create using POST

File: docs/src/main/paradox/docs/delta/api/current/kg-files-api.md [hunk 4]
```diff
@@ -35,109 +32,100 @@ the `prefix` defined on the resource's project (`{project_label}`).
...
 
 Request
-:   @@snip [file.sh](assets/files/file.sh)
+:   @@snip [create-post.sh](assets/files/create-post.sh)
 
 Response
-:   @@snip [file-created.json](assets/files/file-created.json)
+:   @@snip [created-post.json](assets/files/created-post.json)
 
 ## Create a file using PUT
```
Comment (imsdu) [inline docs/src/main/paradox/docs/delta/api/current/kg-files-api.md:40]: Create using PUT

File: docs/src/main/paradox/docs/delta/api/current/kg-files-api.md [hunk 5]
```diff
@@ -35,109 +32,100 @@ the `prefix` defined on the resource's project (`{project_label}`).
...
+... where `{storageId}` selects a specific storage backend where the file will be uploaded. This field is optional. 
+When not specified, the default storage of the project is used.
+
+Note that if the payload contains an @id different from the `{file_id}`, the request will fail.
 
 **Example**
 
 Request
-:   @@snip [file-post-storageid.sh](assets/files/file-post-storageid.sh)
+:   @@snip [crete-put.sh](assets/files/create-put.sh)
```
Comment (imsdu) [inline docs/src/main/paradox/docs/delta/api/current/kg-files-api.md:57]: create-put.sh

**Review by imsdu — CHANGES_REQUESTED** [review]

**Review by imsdu — APPROVED** [review]

**Review by wwajerowicz — APPROVED** [review]

**Pull request merged**